server/src/main/java/com/linecorp/centraldogma/server/internal/api/HttpApiExceptionHandler.java

```diff
@@ -162,6 +167,12 @@ public HttpResponse onServiceException(ServiceRequestContext ctx, Throwable caus
             return newResponse(ctx, HttpStatus.SERVICE_UNAVAILABLE, peeledCause);
         }
 
+        if (peeledCause instanceof RequestTimeoutException) {
+            ctx.setShouldReportUnloggedExceptions(false);
+            return newResponse(ctx, HttpStatus.SERVICE_UNAVAILABLE,
+                               new ApiRequestTimeoutException("Request timed out", peeledCause));
```

Comment:
Question) Is there a reason a separate `ApiRequestTimeoutException` is defined instead of using `RequestTimeoutException` directly?

```suggestion
                               peeledCause);
```

Reply:
`ApiRequestTimeoutException` was added so that the client can deserialize the exception and throw it:
- `RequestTimeoutException` does not have a public constructor to create with a message.
- `CentralDogma` client is designed to deserialize only `CentralDogmaException` types, and I preferred to keep the behavior unchanged.